emitter/src/lib/core/actions/actions.ts

```diff
@@ -2,15 +2,11 @@
  * This class is used as a default action when the user doesn't pass any custom action as an argument
  */
 export class EmitterAction<T = void> {
-  /**
-   * Action type
-   */
-  public static type: string | null = null;
-
   /**
    * Creates EmitterAction instance
    *
    * @param payload - Data to dispatch
+   * @param type - Action type
    */
-  constructor(public payload: T) {}
+  constructor(public payload: T, public type: string | null = null) {}
```

Comment:
No no, `type` should be a static property and should relate to the class, not instance!!!